Fast and Accurate GPU Quantization for Transformers.md

```diff
@@ -265,19 +265,19 @@ The diagrams below depict 32x64 matrices where each numerical value represents t
 Zooming in on the first 16 x 4 elemnts gives a clearer picture of the layout pattern:
 
 #### Row major (CUBLASLT_ORDER_ROW)
-![](_attachments/Pasted%20image%2020230329113112.png)
+<img src="_attachments/row_maj.svg" width="500">
```

Comment:
These are looking _so_ much better. My only hesitation is the white bordering - I don't think it's an issue since people can only read the blog with a white background. But might be worth revisiting if we ever support dark mode